scripts/build_cudaq.sh

```diff
@@ -126,19 +126,18 @@ fi
 # (utils are needed for custom testing tools, e.g. CircuitCheck)
 echo "Preparing CUDA Quantum build with LLVM installation in $LLVM_INSTALL_PREFIX..."
 cmake_args="-G Ninja "$repo_root" \
-  -DCMAKE_COMPILE_WARNING_AS_ERROR=OFF \
+  -DCMAKE_COMPILE_WARNING_AS_ERROR=ON \
   -DCMAKE_INSTALL_PREFIX="$CUDAQ_INSTALL_PREFIX" \
   -DLLVM_DIR="$llvm_lib_dir/cmake/llvm" \
   -DNVQPP_LD_PATH="$NVQPP_LD_PATH" \
   -DCMAKE_BUILD_TYPE=$build_configuration \
   -DCMAKE_EXPORT_COMPILE_COMMANDS=ON \
   -DCUDAQ_ENABLE_PYTHON=TRUE \
-  -DLLVM_BUILD_UTILS=ON \
+  -DCUDAQ_TEST_MOCK_SERVERS=TRUE \
```

Comment:
Are we sure we should do this here? There are no ServerHelpers in the repo currently.

Reply:
And the compilation pipelines are still missing. I'll turn it off, though I am a bit concerned with having code checked in that is not actively tested.